common/script_values/imperia_political_discord_values.txt

```diff
@@ -1254,3 +1241,404 @@ imperia_discord_serf_contribution_value = {
 	}
 	max = imperia_get_peasant_pop_fraction_refined
 }
+
+discord_break_lvl_1_lower = {
+	# An inverted parabola formula.. Yes.
+	# y=-x^2/25+10x-525
+	# y = chance of an event in this range being chosen
+	# x = current discord
+	# Starts at 75 discord (which is not relevant since discord events doesnt pop there), highest chance at 125 discord, stops at 175 discord
+	subtract = {
+		value = {
+			value = var:political_discord
+			multiply = var:political_discord
+		}
+		divide = 25
+	}
+	add = {
+		value = 10
+		multiply = var:political_discord
+	}
+	subtract = 525
+	# min/max to prevent the random_list from freaking out from negative numbers
+	min = 0
+	max = 100
+}
+
+discord_break_lvl_1_upper = {
+	# Inverted parabola formula
+	# y=-x^2/25+14x-1125
+	# Starts at 125 discord, highest chance at 175 discord, stops at 225 discord
+	subtract = {
+		value = {
+			value = var:political_discord
+			multiply = var:political_discord
+		}
+		divide = 25
+	}
+	add = {
+		value = 14
+		multiply = var:political_discord
+	}
+	subtract = 1125
+	min = 0
+	max = 100
+}
+
+discord_break_lvl_2_lower = {
+	# Inverted parabola formula
+	# y=-x^2/25+18x-1925
+	# Starts at 175 discord, highest chance at 225 discord, stops at 275 discord
+	subtract = {
+		value = {
+			value = var:political_discord
+			multiply = var:political_discord
+		}
+		divide = 25
+	}
+	add = {
+		value = 18
+		multiply = var:political_discord
+	}
+	subtract = 1925
+	min = 0
+	max = 100
+}
+
+discord_break_lvl_2_upper = {
+	# Inverted parabola formula
+	# y=-x^2/25+22x-2925
+	# Starts at 225 discord, highest chance at 275 discord, stops at 325 discord
+	subtract = {
+		value = {
+			value = var:political_discord
+			multiply = var:political_discord
+		}
+		divide = 25
+	}
+	add = {
+		value = 22
+		multiply = var:political_discord
+	}
+	subtract = 2925
+	min = 0
+	max = 100
+}
+
+discord_break_lvl_3_lower = {
+	# Inverted parabola formula
+	# y=-x^2/25+26x-4125
+	# Starts at 275 discord, highest chance at 325 discord, stops at 375 discord
+	subtract = {
+		value = {
+			value = var:political_discord
+			multiply = var:political_discord
+		}
+		divide = 25
+	}
+	add = {
+		value = 26
+		multiply = var:political_discord
+	}
+	subtract = 4125
+	min = 0
+	max = 100
+}
+
+discord_break_lvl_3_upper = {
+	# Inverted parabola formula
+	# y=-x^2/25+30x-5525
+	# Starts at 325 discord, highest chance at 375 discord, stops at 425 discord (which is probably not going to be exceeded..?)
+	subtract = {
+		value = {
+			value = var:political_discord
+			multiply = var:political_discord
+		}
+		divide = 25
+	}
+	add = {
+		value = 30
+		multiply = var:political_discord
+	}
+	subtract = 5525
+	min = 0
+	max = 100
+}
+
+discord_option_1_base_value = {
```

Comment:
So its setting a base value for each option in how severe it can get, for the purpose of figuring out how large of a discord loss should be given when selecting that option. I'll add a comment on it.